modules/installation-aws-config-yaml.adoc

```diff
@@ -27,7 +27,8 @@ This sample YAML file is provided for reference only. You must obtain your
 apiVersion: v1beta3
 baseDomain: example.com <1>
 controlPlane: <2>
-- name: master
+- hyperthreading: Enabled
```

Comment:
hyperthreading is Enabled by default and we shouldn't require all install-config.yaml examples to include that.
only the disabled hyperthreading example must include this field being set.

Reply:
@abhinavdahiya, I sent you a longer explanation in a Slack DM. We're handling all of the valid changes to the `install-config.yaml` file at once for each installation type. The installation assemblies are supposed to cover all of the installation tasks, and since this setting has to be set during installation, we can't put it somewhere else.

Reply:
@kalexand-rh When HT is disabled aws instance should be configured to have larger instance size as in:

hyperthreading: Enabled
  name: worker
  platform: {}
  replicas: 2
controlPlane:
  hyperthreading: Disabled
  name: master
  platform:
    aws:
      type: m5.2xlarge
  replicas: 2